cartridges/int_komoju_sfra/cartridge/controllers/KomojuController.js

```diff
@@ -649,40 +649,30 @@ server.post('HandleWebHooksRefund', function (req, res, next) {
 server.post('HandleWebHooksCaptureComplete', function (req, res, next) {
     var Order = require('dw/order/Order');
     var customKomojuSourceLogger = Logger.getLogger('customKomojuSourceLogger', 'customKomojuSourceLogger');
-
     var paymentIdKomoju = JSON.parse(req.body).data.id;
     var body = JSON.parse(req.body);
```

Comment:
Small thing and sorry I noticed it now, but I think we can use the defined `body` here?
```js
    var body = JSON.parse(req.body);
    var paymentIdKomoju = body.data.id;
```